spec/paperclip/has_attached_file_spec.rb

```diff
@@ -34,31 +34,41 @@
       assert_adding_attachment('avatar').defines_callback('before_destroy')
     end
 
-    it 'defines an after_commit callback' do
+    it 'defines the Paperclip-specific callbacks' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.